paymentsheet/src/main/java/com/stripe/android/paymentsheet/state/PaymentElementLoader.kt

```diff
@@ -275,15 +266,6 @@ internal class DefaultPaymentElementLoader @Inject constructor(
         return@runCatching state
     }
 
-    private fun ElementsSession.shouldWarmUpIntegrity(): Boolean = when {
```

Comment:
since we're running an attestation check (that prepares integrity if needed) this is redundant now.